requests-for-comments/2023-17-produchog.md

```diff
@@ -0,0 +1,92 @@
+
+- The current branding was a simple way to get started, but is limiting.
+- ‘ProductHog’ implies what the newsletter is about, while still drawing a connection between it and PostHog.
+- Actions: create website for ProductHog (we own the domain) and launch dedicated social channels.
+
+### Why Substack?
+
+Currently the newsletter is on Mailchimp, but we’ve recently migrated other email ops to [Customer.io](http://Cusomter.io). 
+
+I’m proposing Substack because:
+
+- It’s a popular platform people know and like
+- It would make this feel distinct from other “marketing emails”
+- Allows us to tap into the Substack network effects to aid growth
+
+The downsides of Substack:
+
+- We’ll have less control over branding and appearance
+- **Important:** Substack’s [official embeddable signup form](https://support.substack.com/hc/en-us/articles/360041759232-Can-I-embed-a-signup-form-for-my-publication-) is very limited – it’s not customisable and there’s no public API or any sign they’re going to support one. [Viable workarounds](https://docs.substackapi.com/) may exist, but this is a major downside. The default Substack signup box (I believe) redirects people to Substack’s website as a result. Is this an acceptable trade-off for Substack’s network effects?
+
+If not Substack, we’ll move to [Customer.io](http://Customer.io) as originally planned.
+
+### Who is ProductHog for?
+
+As is currently the case, it’ll be closely aligned to our ICP and target in order of importance:
+
+- CEOs and founders
+- Product engineers
+- Product managers and product owners
+- Growth hackers and product marketers
+
+### Who is ProductHog NOT for?
+
+- Indie hackers (they may find it useful, but it won’t be tailored to their interests)
+- Marketing teams (even when we discuss marketing adjacent topics)
+- Traditional “unengaged” engineers
+- Data scientists
+
+## How to 10x ProductHog
+
+### Make each issue more thematic
+
+Currently, each issue is a bit of a mishmash of different topics and ideas. This provides variety, but not much focus. Curation should still be a core part of the newsletter, but ideally each issue should be more like a magazine, where the core content is themed around a particular topic or idea.
+
+### Double down on curation
+
+In addition to more focused original content, we should become known as a leading source for curated articles on product growth and high-growth startups. This means having more curated articles and (likely) adding categories to organise them more clearly for readers.
+
+### De-emphasise PostHog specific tutorials
+
+Having a high emphasis on content about doing specific things in PostHog will put off people who aren’t already PostHog users. Our goal with ProductHog is to raise awareness of PostHog and promote our values, rather than inform existing users of new content / how to stuff. We should launch a new email list / create a new way to alert PostHog users to new tutorials.
```

Comment:
I'm fine with not including tutorials, but I feel like launching another newsletter just for tutorials will be overkill. Let's just focus on making ProductHog amazing and we can figure out tutorial distribution later.

Reply:
Fair. I think the solution here is about making tutorials more visible in the product, which I think Cory has been working on as part of PostHog 3000.